app/src/main/java/org/oppia/app/player/state/itemviewmodel/DragDropInteractionContentViewModel.kt

```diff
@@ -1,9 +1,20 @@
 package org.oppia.app.player.state.itemviewmodel
 
+import androidx.recyclerview.widget.RecyclerView
 import org.oppia.app.model.StringList
 import org.oppia.app.viewmodel.ObservableViewModel
 
 /** [ObservableViewModel] for DragDropSortInput values. */
 class DragDropInteractionContentViewModel(
-  var htmlContent: StringList
-) : ObservableViewModel()
+  var htmlContent: StringList, var itemIndex: Int, var listSize: Int,
```

Comment:
Suggest writing this in multiple lines for better code clarity.
Also, can be make these `var` as `val`
```
  val htmlContent: StringList,
  val itemIndex: Int,
  val listSize: Int,
  private val dragAndDropSortInteractionViewModel: DragAndDropSortInteractionViewModel
```

Reply:
done